pasturekey.md

```
@@ -2129,3 +2151,40 @@ in the stats list, one for each year.
+----------| -----------|-----------|--------
+/snapshot | TSDM       | median    | smoothed median from multiple capture dates
+/snapshot | TSDM       | captured_median<sup>*</sup> | median from single capture date
+/snapshot | TSDM green | median    | smoothed median from multiple capture dates
+/snapshot | TSDM dead  | median    | smoothed median from multiple capture dates
+/snapshot | FC         | median    | smoothed median from multiple capture dates
+/gettsdmstats | TSDM   | median    | smoothed median from multiple capture dates
+/gettsdmstats | TSDM   | captured_median<sup>*</sup> | median from single capture date
+/gettsdmgreenstats | TSDM green | median | smoothed median from multiple capture dates
+/gettsdmdeadstats  | TSDM dead  | median | smoothed median from multiple capture dates
+/getfcstats | FC       | median    | smoothed median from multiple capture dates
+/subpaddock | all      | median    | median from the requested product - see the notes in the /subpaddock endpoint for details |
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)